core/trino-main/src/main/java/io/trino/operator/DriverContext.java

```diff
@@ -346,7 +346,7 @@ public DriverStats getDriverStats()
         if (inputOperator != null) {
             physicalInputDataSize = inputOperator.getPhysicalInputDataSize();
             physicalInputPositions = inputOperator.getPhysicalInputPositions();
-            physicalInputReadTime = inputOperator.getAddInputWall();
+            physicalInputReadTime = inputOperator.getPhysicalInputReadTime();
 
             internalNetworkInputDataSize = inputOperator.getInternalNetworkInputDataSize();
             internalNetworkInputPositions = inputOperator.getInternalNetworkInputPositions();
```

Comment:
I don't think `internalNetworkInputReadTime = inputOperator.getAddInputWall();` is ever used? Could we just remove it and add a TODO (+ github issue)? Alternatively you could just fix it

Reply:
Added a commit to remove internalNetworkInputReadTime